ir-engine/ir-engine
# IR-3663 video controls size and responsiveness

## Summary
fixing the mediacontrols button scaling and reactivity (play/pause button icons update). Also managed to set scale information properly depending video canvas size and transform scale (chooses smaller of combined size for controls scale)


![image](https://github.com/user-attachments/assets/bdb41372-2ae6-480c-be9f-1ab1401380b9)
are you not entertained?

## References
[https://tsu.atlassian.net/browse/IR-3663](https://tsu.atlassian.net/browse/IR-3663)

## QA Steps
put a video in your scene, and update the transform's scale and the video canvas size as you see fit. The media controls in the published scene should be roughly 10% of the size of the smaller dimension of width or height
